package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "chat",
+  "version": "1.0.0",
+  "description": "chat",
+  "main": "index.js",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "start": "PORT=3000 parcel src/pages/signin.hbs --no-cache --open",
+    "build": "parcel build src/pages/signin.hbs --no-cache --public-url ./",
```

Лучше использовать "wildcard" на случай если сборщику не всегда удастся самому разрешить все зависимости между шаблонами
```suggestion
    "start": "PORT=3000 parcel src/pages/*.hbs --no-cache --open",
    "build": "parcel build src/pages/*.hbs --no-cache --public-url ./",
```

---

Поправил